tests/data/profile_stability/rhel8/stig.profile

```diff
@@ -69,6 +69,7 @@ selections:
 - accounts_umask_etc_profile
 - accounts_umask_interactive_users
 - accounts_user_dot_no_world_writable_programs
+- accounts_users_home_files_permissions
```

Comment:
There is also the RHEL 8 STIG GUI profile, which extends the STIG profile, so to make the profile stability test pass you need to add the rule ID also to the profile stability test image for the STIG GUI profile in tests/data/profile_stability/rhel8/stig_gui.profile.